packages/lab/src/radio-button/RadioButton.css

```diff
@@ -1,57 +1,35 @@
-/* Styles applied to root component  */
-.salt-density-medium {
-  --radioButton-label-marginTop: 6px;
-  --radioButton-row-marginRight: 24px;
-}
-
-.salt-density-high {
-  --radioButton-label-marginTop: 3px;
-  --radioButton-row-marginRight: 12px;
-}
-
-.salt-density-low {
-  --radioButton-label-marginTop: 9px;
-  --radioButton-row-marginRight: 36px;
-}
-
-.salt-density-touch {
-  --radioButton-label-marginTop: 12px;
-  --radioButton-row-marginRight: 48px;
-}
-
+/* Styles applied to RadioButton container */
 .saltRadioButton {
+  display: flex;
+  gap: calc(var(--salt-size-unit) * 0.75);
+  position: relative;
+  user-select: none;
   cursor: pointer;
-  display: block;
-}
-
-/* Styles applied to root component on hover if `disabled={true}` or FormField disabled */
-.saltRadioButton-disabled,
-.saltFormField-disabled .saltRadioButton {
-  cursor: var(--salt-selectable-cursor-disabled);
 }
 
+/* Styles applied when RadioButton is disabled */
 .saltRadioButton-disabled {
-  --saltControlLabel-color: var(--salt-text-primary-foreground-disabled);
-
+  color: var(--salt-text-primary-foreground-disabled);
+  cursor: var(--salt-selectable-cursor-disabled);
   pointer-events: visible;
 }
 
-/* Styles applied to container of label and Radio component */
-.saltRadioButton-labelContainer {
-  display: inline-flex;
-  align-items: flex-start;
-  margin-left: 0;
-  margin-right: 0;
-  margin-top: var(--saltRadioButton-label-marginTop, var(--radioButton-label-marginTop));
-  vertical-align: baseline;
-}
-
-.saltRadioButton-labelContainer .saltRadioButton {
-  display: inline-flex;
-  align-content: flex-start;
-  vertical-align: baseline;
-}
-
-.saltFormGroup-row > .saltRadioButton {
-  margin-right: var(--saltFormGroup-row-marginRight, var(--radioButton-row-marginRight));
+/* Styles applied to input component */
+.saltRadioButton-input {
+  cursor: inherit;
```

Comment:
this means the label is clickable and even in its padding space, is that desired?